content/docs/add-react-to-a-website.md

```diff
@@ -1,202 +1,201 @@
 ---
 id: add-react-to-a-website
-title: Add React to a Website
+title: Ajouter React à un site web
 permalink: docs/add-react-to-a-website.html
 redirect_from:
   - "docs/add-react-to-an-existing-app.html"
 prev: getting-started.html
 next: create-a-new-react-app.html
 ---
 
-Use as little or as much React as you need.
+Utiliser React en fonction de vos besoins
 
-React has been designed from the start for gradual adoption, and **you can use as little or as much React as you need**. Perhaps you only want to add some "sprinkles of interactivity" to an existing page. React components are a great way to do that.
+Depuis sa création, React s'adapte en fonction de vos besoins, ainsi vous pouvez l'utiliser autant que vous le souhaitez, ou simplement ajouter un peu d'interactivité à une page existante. Les composants React sont une bonne solution pour faire ça.
 
-The majority of websites aren't, and don't need to be, single-page apps. With **a few lines of code and no build tooling**, try React in a small part of your website. You can then either gradually expand its presence, or keep it contained to a few dynamic widgets.
+La majorité des sites web ne sont pas, et n'ont pas besoin d'être, des applications web monopage (en anglais Single Page Application ou plus simplement SPA, NdT). Sans outils ou configurations supplémentaires, mais avec simplement quelques lignes de code, vous pouvez essayer React dans une petite partie de votre site web. Vous pourrez par la suite ajouter des composants supplémentaires ou garder React dans quelques éléments dynamiques.
```

Comment:
```suggestion
La majorité des sites web ne sont pas, et n'ont pas besoin d'être, des applis web monopages *(Single-Page Apps ou plus simplement SPA, NdT)*. **Avec juste quelques lignes de code et sans outillage particulier**, vous pouvez essayer React sur une petite partie de votre site web. Vous pourrez par la suite ajouter des composants supplémentaires ou limiter React à quelques éléments dynamiques.
```

- Formatage adapté de la NdT
- Retour du formatage VO, qui n'a rien d'anecdotique
- Fluidification

Reply:
Je suis pas très partisan de "applis" ou meme de "apps". Application est quand meme le terme officiel en anglais et en français. Du coup je serais plus partant pour garder ta correction mais avec "application".

```suggestion
La majorité des sites web ne sont pas, et n'ont pas besoin d'être, des applications web monopages *(Single-Page Application ou plus simplement SPA, NdT)*. **Avec juste quelques lignes de code et sans outillage particulier**, vous pouvez essayer React sur une petite partie de votre site web. Vous pourrez par la suite ajouter des composants supplémentaires ou limiter React à quelques éléments dynamiques.
```

Reply:
@Nab69 le mot "appli" n'existe pas plus que "démo", et pourtant ils sont en usage courant, y compris à l'écrit, dans des documents officiels, etc. En pratique, la VO utilise parfois “application” (qu'on traduira tel quel), parfois “app” (qu'on traduira en “appli”), pour préserver le rythme et le ton de la VO.

Reply:
@tdd je suis pas non plus un grand fan de "démo" mais au moins celui-ci apparait dans des dictionnaires en ligne.
C'est peut être l'occasion de faire évoluer une mauvaise traduction en utilisant les bons termes ?
D'ailleurs je pense @sheplu partage mon avis sur ce point.

Reply:
Tu imagines bien que les dicos en ligne ne pourront jamais suivre la réalité du terrain technologique. Et je ne suis pas sûr de l'intention derrière les termes « mauvaise traduction ».

Quoi qu'il en soit, je ne vais pas non plus me battre pour ce qui est, *in fine*, un détail qui ne corrompt pas le sens du texte, mais en altère simplement le rythme. Je t'invite donc à valider ta suggestion qui reprenait mon texte à ce terme près.

Reply:
Il n'y a pas de mauvaises intentions derrière ça, juste des traductions 😉 
Par ailleurs on peut quand même en débattre.

Reply:
Je pense qu'on est arrivé au bout de ce débat ; prends ta dernière suggestion qui retient mon texte à « application » près, ça ira bien.